WordPress/WordPressScreenshotGeneration/WordPressScreenshotGeneration.swift

```diff
@@ -173,6 +178,61 @@ class WordPressScreenshotGeneration: XCTestCase {
             notNowButton.tap()
         }
 
-        snapshot("5-Notifications")
+        snapshot("3-Notifications")
+    }
+
+    private func screenshotPost(withSlug slug: String, called screenshotName: String, withKeyboard: Bool = false) {
+
+        let app = XCUIApplication()
+
+        let cell = app.tables.cells[slug]
+        waitForElementToExist(element: cell)
+
+        scrollElementIntoView(element: cell, within: app.tables["PostsTable"])
+        cell.tap()
+
+        let editorNavigationBar = app.navigationBars["Azctec Editor Navigation Bar"]
+        waitForElementToExist(element: editorNavigationBar)
+
+        if !withKeyboard {
+            app.textViews["aztec-editor-title"].tap(withNumberOfTaps: 1, numberOfTouches: 5)
+        }
+
+        sleep(imagesWaitTime) // wait for post images to load
+        // The title field gets focus automatically
+        snapshot(screenshotName)
+
+        editorNavigationBar.buttons["Close"].tap()
+        // Dismiss Unsaved Changes Alert if it shows up
+        if app.sheets.element(boundBy: 0).exists {
+            // Tap discard
+            app.sheets.element(boundBy: 0).buttons.element(boundBy: 1).tap()
+        }
+    }
+
+    private func gutenScreenshot() {
+
+        let app = XCUIApplication()
+
+        app.tables["Blog Details Table"]
+            .cells["Site Pages Row"].tap() // tap Site Pages
+        waitForElementToExist(element: app.tables["PagesTable"])
+
+        // Switch the filter to drafts
+        app.buttons["drafts"].tap()
+
+        app.tables.cells["gutenpage"].tap()
+        sleep(10)
+        snapshot("7-gutenpage")
+
+//        app.buttons["editor-close-button"].tap()
```

Comment:
Why is this commented?

Reply:
Because I have a bad habit of keeping code around in comments that I think might be needed later 😞 

Removed!